ban/tests/test_validators.py

```diff
@@ -198,21 +221,21 @@ def test_cannot_create_postcode_with_code_shorter_than_5_chars(session):
     municipality = MunicipalityFactory(insee='12345')
     validator = models.PostCode.validator(code="3131", name="Montbrun-Bocage",
                                           municipality=municipality)
-    assert validator.errors['code'] == 'Invalid postcode: `3131`'
+    assert validator.errors['code'] == '`3131` should be minimum 5 characters'
 
 
 def test_cannot_create_postcode_with_code_bigger_than_5_chars(session):
     municipality = MunicipalityFactory(insee='12345')
     validator = models.PostCode.validator(code="313100", name="Montbrun",
                                           municipality=municipality)
-    assert validator.errors['code'] == 'Invalid postcode: `313100`'
+    assert validator.errors['code'] == '`313100` should be maximum 5 characters'
 
 
 def test_cannot_create_postcode_with_code_non_digit(session):
     municipality = MunicipalityFactory(insee='12345')
     validator = models.PostCode.validator(code="2A000", name="Montbrun-Bocage",
                                           municipality=municipality)
-    assert validator.errors['code'] == 'Invalid postcode: `2A000`'
+    assert validator.errors['code'].startswith('Wrong format. Value should')
```

Comment:
Good catch! The message was actually `Wrong format. Value should match: `re.compile('\\d*')`` :p
Fixed, thanks :)